tests/api.c

```diff
@@ -37336,7 +37336,7 @@ static void test_wolfssl_EVP_aes_gcm_zeroLen()
 
     AssertIntEQ(1, EVP_EncryptInit_ex(en, EVP_aes_256_gcm(), NULL, key, iv));
     AssertIntEQ(1, EVP_CIPHER_CTX_ctrl(en, EVP_CTRL_GCM_SET_IVLEN, ivSz, NULL));
-    AssertIntEQ(1, EVP_EncryptUpdate(en, NULL, &ciphertxtSz , plaintxt, plaintxtSz));
+    AssertIntEQ(1, EVP_EncryptUpdate(en, ciphertxt, &ciphertxtSz , plaintxt, plaintxtSz));
```

Comment:
Why provide an output buffer here but not for `EVP_DecryptUpdate`?

Reply:
This is for additional test case that size is zero and the pointer is not NULL. It is reported by the user.
ctx->gcmBuffer is allocated  in EVP_CipherUpdate_GCM if input pointer is not NULL regardless of the size.